aws/resource_aws_lb_listener.go

```diff
@@ -25,6 +25,10 @@ func resourceAwsLbListener() *schema.Resource {
 			State: schema.ImportStatePassthrough,
 		},
 
+		Timeouts: &schema.ResourceTimeout{
```

Comment:
If we do decide to add user configurable timeouts for the resource (I would recommend against in this situation), it should be documented in the resource documentation under a `## Timeouts` header.